src/v2/Apps/Conversation/Components/__tests__/ConfirmArtwork.jest.tsx

```diff
@@ -187,4 +232,24 @@ describe("Artwork editions", () => {
       expect(radios[1]).toBeChecked()
     })
   })
+
+  it("Correctly displays availability for all relevant cases", async () => {
+    renderWithRelay(mockActionableEditionCases)
+
+    await waitFor(() => {
+      const radios = screen.getAllByRole("radio")
+      expect(radios).toHaveLength(4)
+
+      // Test unavailable edition
+      const lastRadio = radios.pop()
+      expect(lastRadio).toHaveAttribute("disabled")
+      expect(lastRadio).toHaveTextContent("Unavailable")
+
+      // Make sure the rest are not disabled
+      for (const radio of radios) {
+        expect(radio).toBeEnabled()
```

Comment:
For whatever reason the linter wants to change `.not.toHaveAttribute("disabled")`, to `.toBeEnabled()`, but they're not equivalent.
So, added the following line `.not.toHaveTextContent("Unavailable")` for safety.